rollup.config.js

```diff
@@ -4,9 +4,13 @@ import pkg from './package.json'
 export default {
     input: 'src/index.js',
     output: {
+        file: 'c3.js',
         name: 'c3',
         format: 'umd',
-        banner: `/* @license C3.js v${pkg.version} | (c) C3 Team and other contributors | http://c3js.org/ */`
+        banner: `/* @license C3.js v${pkg.version} | (c) C3 Team and other contributors | http://c3js.org/ */`,
+        globals:{
+            d3:'d3'
```

Comment:
This will list `d3` as a dependency in the top of the UMD build, instead of requiring `d3` from `ChartInternal`